Add missing translation keys for the Capes options Screen

moehreag opened the pull request:
### First:
What the titel says.

### Second:
The Capes Options Screen is maybe still being rendered a bit toooo big (about 30px in the height?, the width should be good then, this might also only apply to my 4:3 screens, sadly I don't have any other ones to test it with.).

### Third:
Somehow, though hardcoded, you're not being stated as the Creator of the Osmium logo cape. Here, have a screenshot:
![Screenshot_20210921_202121](https://user-images.githubusercontent.com/85403855/134226126-4c826d04-4c97-416d-84c7-7e76e2fd292f.png)

Otherwise, keep up your great work! Some day, even the big Clients will have to port to Fabric so your mod will already be there against them!

moehreag committed:
Add missing translation keys

Intro-Dev commented:
Translations look great!

As for the other issues, I have fixed the issues with creator names not showing up in an uncommitted change, and I am addressing the issues with gui scaling before the next update release.

moehreag commented:
Thanks! Some Day, with much much time, I will maybe PR a de_de.json Lang file...

Intro-Dev commented:
That would be great!

moehreag commented:
> Translations look great!
> 
> As for the other issues, I have fixed the issues with creator names not showing up in an uncommitted change, and I am addressing the issues with gui scaling before the next update release.

Have you already try to fix the GUI scaling? If yes (what I assume since the layout has changed since my last test), it sadly does not work correctly. 
![2021-09-22_19 01 18](https://user-images.githubusercontent.com/85403855/134390453-8ca9d773-a02a-4a03-b3cf-c4e9af15443b.jpg) (_I took the screenshot with my collection of mods, if you want a mod list, look [here](http://v2202010130453128524.happysrv.de/axoclient/files/distribution.json)_)

If you didn't change anything in there, just please ignore this comment.
